lib/ar_lazy_preload/active_record/relation.rb

```diff
@@ -47,6 +52,10 @@ def lazy_preload_values
 
     private
 
+    def _ar_lazy_preload_records
+      @records || []
```

Comment:
I guess we don't need a fallback here cause when `@records` are not available we should handle this case anyway